apps/api/internal/common/secrets.go

```diff
@@ -32,21 +35,34 @@ func GetSecrets() *Secrets {
 }
 
 func loadOrGenerateSecrets() *Secrets {
-
 	err := godotenv.Load(secretsFilePath)
 	if err != nil {
 		log.Println("[WARN] No .env file found, will generate secrets if missing")
 	}
 
 	jwtSecret := getOrGenerateSecret("JWT_SECRET")
 	encryptionKey := getOrGenerateSecret("ENCRYPTION_KEY")
+	adminUsernameCredential, _ := getWithoutGenerateSecret("ADMIN_USERNAME_CREDENTIAL")
```

Comment:
Could you update the documentation about these environment variable changes? Please specify if they are optional or required